modules/katamari/src/main/ts/ephox/katamari/api/Result.ts

```diff
@@ -103,7 +97,7 @@ const value = <T, E = any>(o: T): Result<T, E> => {
   };
 };
 
-const error = <T = any, E = any>(message: E): Result<T, E> => {
+const error = <T = never, E = any>(message: E): Result<T, E> => {
```

Comment:
Awesome to have this `any` removed — we should try to get the other one on this line in the next slack time PR